pkg/cli/cmds/server.go

```diff
@@ -489,6 +490,11 @@ var ServerFlags = []cli.Flag{
 		Usage:       "(experimental/components) Disable running etcd",
 		Destination: &ServerConfig.DisableETCD,
 	},
+	&cli.BoolFlag{
+		Name:        "embedded-registry",
+		Usage:       "(experimental/components) Enable embedded distributed container registry; requires use of embedded containerd",
```

Comment:
When you say embedded containterd, do you mean k3s managed containerd or is there a separate option for using embedded containerd?